packages/client/src/containers/CarouselWrapper/MostPopular.jsx

```diff
@@ -12,7 +12,7 @@ export const MostPoppular = () => {
         const data = await response.json();
         setPopularMovies(data.slice(1, 12));
```

Comment:
@Kositthai Can you please confirm the structure of the `data` object here? I am getting an error here.